pandas/tests/copy_view/test_methods.py

```diff
@@ -52,6 +52,24 @@ def test_copy_shallow(using_copy_on_write):
 # DataFrame methods returning new DataFrame using shallow copy
 
 
+def test_drop_on_column(using_copy_on_write):
+    df = DataFrame(
+        {"a": [1, 2, 3], "b": [4, 5, 6], "c": [0.1, 0.2, 0.3]}, index=[10, 11, 12]
```

Comment:
```suggestion
        {"a": [1, 2, 3], "b": [4, 5, 6], "c": [0.1, 0.2, 0.3]}
```

(I used a custom index for the reset_index test, but for the drop test, a custom index shouldn't matter)